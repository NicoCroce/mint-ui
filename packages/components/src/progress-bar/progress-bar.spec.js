import * as React from "react";
import { render, cleanup } from "@testing-library/react";
import ProgressBar from "./";

export const TestProgressBar = () => {
	
	return (
		<ProgressBar
			title="Life profile"
			progress={50}
			showProgressPercent={true}
		/>
	);
};

afterEach(cleanup);

describe("ProgressBar tests", () => {
	test("renders correcty", () => {
		const { asFragment } = render(<TestProgressBar />);
		expect(asFragment()).toMatchSnapshot();
	});
	
});
