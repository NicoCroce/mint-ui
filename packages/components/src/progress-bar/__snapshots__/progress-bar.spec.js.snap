// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar tests renders correcty 1`] = `
<DocumentFragment>
  <div
    class="progress-bar__ProgressBarWrapper-sciksu-4 juHzRM"
  >
    <div
      class="progress-bar__ProgressTitleWrapper-sciksu-0 jIMkJk"
    >
      <span
        class="progress-bar__ProgressBarLeftText-sciksu-2 jQjCkV"
      >
        Life profile
      </span>
      <span
        class="progress-bar__ProgressBarRightText-sciksu-3 hKMZbi"
      >
        <span>
          50% Complete
        </span>
      </span>
    </div>
    <div
      class="progress-bar__ProgressBox-sciksu-5 ldxOPT"
    >
      <div
        class="progress-bar__ProgressLine-sciksu-1 lkboAd"
      />
    </div>
  </div>
</DocumentFragment>
`;
